tests/test_go.py

```diff
@@ -85,3 +85,24 @@ def test_base_PATH_present(auto_container, container):
         [0], "echo $PATH"
     ).stdout.strip()
     assert path_in_base_container in path_in_go_container
+
+
+@pytest.mark.parametrize(
+    "container_per_test, container_git_clone",
+    [
+        (GO_1_18_CONTAINER, 
+        GitRepositoryBuild(
+            repository_url="https://github.com/Code-Hex/go-generics-cache.git",
+            repository_tag="v1.0.1",
+            build_command="go test ./...",
+        ).to_pytest_param()),
```

Comment:
If you wrap this whole thing in a `pytest.param()`, then you should be able to give it a custom `id` with a more descriptive name.

Reply:
Oh yes, that works nicely. PR amended.